x-pack/plugins/ml/server/saved_objects/initialization/initialization.ts

```diff
@@ -68,9 +68,11 @@ export function jobSavedObjectsInitializationFactory(
   }
 
   async function _needsInitializing(savedObjectsClient: SavedObjectsClientContract) {
-    if (await _jobSavedObjectsExist(savedObjectsClient)) {
+    if (false && (await _jobSavedObjectsExist(savedObjectsClient))) {
```

Comment:
Disabling this check for now, just in case we decide that running the initialisation process on every kibana start up is overkill and wish to revert it to only running if no saved objects are found.

Reply:
Based on what are we going to make a decision? Maybe we should measure how long the check takes and report this metric to the telemetry cluster?